devtools/gradle/src/functionalTest/resources/dotenv-config-java-module/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,21 @@
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @ConfigProperty(name = "greeting.message")
```

Comment:
Where is this property defined?

Reply:
in the `.env` file: https://github.com/quarkusio/quarkus/pull/9778/files#diff-777b2fd663200ce61ba6c7c302a48207

Reply:
Ah, so GREETING_MESSAGE is turned into greeting.message in our config?

Reply:
Exactly